doc/Development_Documentation/02_MVC/02_Template/02_Template_Extensions/README.md

```diff
@@ -0,0 +1,288 @@
+# Twig Extensions
+
+## Introduction
+
+Twig Extensions are functions, filters and more that offer special functionality to increase usability of view scripts. 
+
+Following an overview of some [Twig Extensions](https://twig.symfony.com/doc/2.x/#reference):   
+- `render` 
+- `render_esi` 
+- `controller` 
+- `asset` 
+- `csrf_token`  
+- `path` 
+- `url` 
+- `absolute_url` 
+- `translator` 
+- `trans`
+- `url` 
+- `relative_path` 
+
+  
+In addition to the standard Twig extensions, Pimcore adds some additional powerful extensions. 
+  
+## Pimcore Twig Extensions
+
+All Twig extension functions are described below in detail, the following tables give just a short overview of all available extensions.
+
+| Extension                                | Description                                                     |
+|------------------------------------------|-----------------------------------------------------------------|
+| `pimcore_action()`        | (Deprecated) Call an arbitrary action and renders the respective template  |
+| `pimcore_cache()`         | Simple in-template caching functionality                                         |
+| `pimcore_device()`        | Helps implementing adaptive designs.                                             |
+| `pimcoreglossary()`       | Extension to control the glossary engine                                         |
+| `pimcore_placeholder()`   | Adding and embedding custom placeholders, e.g. for special header tags, etc.     |
+| `pimcore_head_link()`     | Embeding / managing referenced stylesheets (alternative to `assets()`)           |
+| `pimcore_head_meta()`     | Managing your \<meta\> elements in your HTML document                            |
+| `pimcore_head_script()`   | Managing your <scripts> elements                                                 |
+| `pimcore_head_style()`    | Managing inline styles (pendant to `headLink()` for inline styles)               |
+| `pimcore_head_title()`    | Create and store the HTML document's `<title>` for later retrieval and output    |
+| `pimcore_inc()`           | Use this function to directly include a Pimcore document.                        |
+| `pimcore_inline_script`   | Managing inline scripts (pendant to `headScript()` for inline scripts)           |
+| `pimcore_build_nav()`, `pimcore_render_nav()`, `pimcore_nav_renderer()`   | Embed and build navigations based on the document structure                             |
+| `pimcore_url()`           | An alternative to `url()` and `path()` with the building behavior of Pimcore 4   |
```

Comment:
`with the building behavior of Pimcore 4` is this relevant?

Reply:
kicked it